### DISCUSSION TITLE
Difference between `extend_line_below` and `select_line_below`?

### DISCUSSION THREAD

**Discussion by RoloEdits** [helix-editor/helix#10013] [Q&A] [answered]
Both select the line if not already selected, and if selected will select the line below. What is the crux that differentiates them?

**Comment by kirawi** [comment, marked as answer]
It addresses this: https://github.com/helix-editor/helix/issues/9079

It can shrink the selection if you go the opposite direction (e.g. if you `x` one too many times, you can go shrink it).

**Reply by RoloEdits** [reply]
Thanks. As far as I can see, this is a superset of the previous x key? Is there a need to have both?

**Reply by chtenb** [reply]
The older command was kept for backward compatibility I think, probably many people are used to it. But I use the newer command too (https://github.com/helix-editor/helix/pull/9080#issuecomment-2008964831)
